test/ios_application_test.sh

```diff
@@ -147,6 +147,11 @@ EOF
 
   cat > app/fmwk.framework/resource.txt <<EOF
 Dummy resource
+EOF
+
+  mkdir -p app/fmwk.framework/fmwk.bundle
+  cat > app/fmwk.framework/fmwk.bundle/Some.plist <<EOF
+Dummy plist
```

Comment:
```
<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
<plist version="1.0">
<dict>
	<key>Foo</key>
	<string>Bar</string>
</dict>
</plist>
```

because if it was processed, I'll bet things would error since it isn't a valid plist?

Reply:
The tests run fine for me, I was a bit surprised too. I can replace it with this anyways though

Reply:
Without your fix, does it work?  i.e. - if the file is processed by accident, what happens then?  If it still passes, then either the test isn't testing what we think it is, or a bogus file slips thru unprocessed meaning the test isn't ensuring it processed.

Reply:
Yep it fails correctly without it

Reply:
It looks like this is expected, plutil happily converts a file like this:

```
% echo "hi" > foo.txt
% plutil -convert binary1 foo.txt
% file foo.txt
foo.txt: Apple binary property list
% cat foo.txt
bplist00Rhi
```

I'll still make the change to make it make more sense

Reply:
Actually there is this same case above for the framework's Info.plist, so should we leave this one or change both?

Reply:
just keep this one, leave the other one as is

Reply:
what I meant was to not modify the other tests, and make the changes that @thomasvl mentioned for this test

Reply:
Done